src/Bicep.LangServer.IntegrationTests/Helpers/IntegrationTestHelper.cs

```diff
@@ -19,13 +19,14 @@
 using Bicep.LanguageServer.Utils;
 using System.Collections.Generic;
 using Bicep.Core.FileSystem;
-using Bicep.Core.UnitTests.FileSystem;
 using Bicep.Core.Navigation;
 
 namespace Bicep.LangServer.IntegrationTests
 {
     public static class IntegrationTestHelper
     {
+        private const int DefaultTimeout = 20000;
```

Comment:
>20000 [](start = 43, length = 5)

When running all the tests in parallel in VS, I would occasionally get a timeout that goes away on retry. Increased it to 20s from 10s.

Reply:
This is not applicable anymore.